templates/zerver/help/restrict-permissions-of-new-members.md

```diff
@@ -22,6 +22,8 @@ Currently, the following actions support limiting access to full members.
 
 ### Set waiting period for new members
 
+{!owner-only.md!}
+
```

Comment:
Have added this here, because the page also mentions that "these settings can be configured" to restrict new members, so I am not sure whether the restriction mentioning "Only owners can ..." should be at the top of page, as these settings can be changes by admins also, but the `waiting period threshold` cannot.

Reply:
Since this page has only one set of instructions, and they are just for owners, let's just put an `{!owner-only.md!}` at the top of the page. Thanks!

Reply:
Updated.